src/coroutine.js

```diff
@@ -1,25 +1,21 @@
+import Action from './Action'
+
 export default function (resolve, iterator, promise) {
 	new Coroutine(resolve, iterator, promise).run()
+	// taskQueue.add(new Coroutine(resolve, iterator, promise))
```

Comment:
Ok, here are representative runs of the perf tests  to compare with the [latest results](https://github.com/briancavalier/creed/blob/master/perf/README.md).  I used the 1.0.2 release version and just changed _runAction to _when, so it's apples to apples.

doxbee:

```
callbacks-baseline.js                     107       30.80
promises-creed-generator.js               201       33.73
promises-creed-algebraic.js               243       47.14
promises-creed.js                         249       47.67
promises-bluebird-generator.js            254       37.50
```

parallel:

```
promises-creed.js                        433      118.45
promises-creed-generator.js              463      116.56
promises-bluebird.js                     501      101.12
promises-bluebird-generator.js           541      103.12
```

doxbee-errors:

```
callbacks-baseline.js                     131       30.23
promises-creed-generator.js               228       33.46
promises-creed.js                         268       49.74
promises-bluebird-generator.js            273       36.66
promises-creed-algebraic.js               277       49.68
callbacks-caolan-async-waterfall.js       279       48.87
```

I'm rather surprised, honestly, that them seem to be roughly equivalent.